packages/client/test/test-utils/fake/FakeBrubeckNode.ts

```diff
@@ -38,6 +38,17 @@ class FakeNetworkNodeStub implements NetworkNodeStub {
         this.subsciptions.delete(streamPartId)
     }
 
+    async subscribeAndWaitForJoin(streamPartId: StreamPartID, _timeout?: number): Promise<number> {
+        this.subsciptions.add(streamPartId)
+        return 1
```

Comment:
There's probably a way to pull the number of other nodes in that stream part. @teogeb do you know?

Reply:
Check the logic in in `publish()`:
```
this.node.activeNodes.getNodes()
            .forEach(async (n) => {
                const networkNode = await n.getNode()
                if (networkNode.subsciptions.has(msg.getStreamPartID())) {
```

If we reuse these lines in two places, let's put these lines into a helper method used by `publish` and this `subscribeAndWaitForJoin`.